UserInterface.adoc

```diff
@@ -80,52 +128,368 @@ fundamental A: If it does behave that way now, I'd think it would be wise to
 make it not behave that way in the future.
 ////
 
-=== Navigation Panel
-
-The left side of the screen can be used to change what view the content panel is
-actively presenting.
-The side panel is used to navigate between different parts of the user
-interface.
-From here you can change the view in the main panel to:
-
-* Part settings
-* Part Grid - the active part id
-* Browser
-* Mixer
-* Kits
-* Kit Grid - the active kit id
-* Macro Learn
-* Effects
-* Add synth
-* Add synth Voice Grid - the active voice id
-* Sub synth
-* Pad synth
-
-For all of the grid selectors you can change the active id with the left mouse
-button or you can enable/disable the selected ID via the middle mouse button.
+==== Navigation Panel
+
+The navigation panel is situated on the left side of the Zyn-Fusion UI and, much like the
+status panel, is always present there.
+
+It is used to change what the main panel is actively presenting.
+The options, from top to bottom, include:
+
+* PART SETTINGS
+* Part grid
+* BROWSER
+* MIXER
+* KITS
+* Kit grid
+* MACRO LEARN
+* EFFECTS
+* The ADDSynth panel
+* The ADDSynth voice grid
+* The SUBSynth panel
+* The PADSynth panel
+
+Each grid houses multiple elements with the same functionality,
+and has one lit-up square, which is the currently displayed element.
+Each active element has a border around it.
+*To view an element in the grid, left-mouse click it.*
+*To enable an element in the grid, middle-mouse click it.*
+
+// TODO: Capitalize each word?
+==== Main Panel (Introduction)
+
+The main panel is where most of the action happens.
+
+The scope of that is, understandably, very large, so it warrants
+<<_main_panel, its own section in this part of the manual, visible below>>.
+
+==== Virtual keyboard panel
+
+The virtual keyboard is located at the bottom of the Zyn-Fusion interface.
+
+// TODO
+
+Modulation wheel::
+    Desc
+
+Keyboard::
+    Desc
+
+VELOCITY::
```

Comment:
Same as the last comment, let the tooltips do the work for you :)

Reply:
Oh, sure! Should I even discuss UI elements which are explained on their own?

Reply:
Pretty much. That and the tutorials were an explicit effort in terms of providing users with a motivation for why they'd want to utilize parts of the UI as well.